jenkins/scripts/select-compiler.sh

```diff
@@ -84,6 +84,12 @@ elif [ "$SELECT_ARCH" = "AIXPPC" ]; then
         echo "Compiler set to 6.3"
 	return
       else
+	# TODO(@sam-github) does now work, LIBPATH overrides the compiled-in
+	# libpaths of all binaries, but we need different libpaths for the
+	# build tools (for example, python), and for the compiled binaries
+	# (node and intermediate tools). Note that 8.x will be EOL at end of
+	# 2019, so there is little reason to get this working, and it should be
+	# removed in 2020.
```

Comment:
Are there any plans to run any of the 8.x jobs on AIX 7.2? It might be easier/simpler to keep those on the 6.1 boxes if it's only until the end of the year.

Reply:
no, but I thought the text might be useful if anybody ever did try. removed it.